assets/locales/en/translation.json

```diff
@@ -68,13 +68,13 @@
   "ImportDataTab": {
     "Sidebar": {
       "ImportDataNavSteps": {
-        "Select File": "Select File",
+        "{{stepNumber}}. Select File": "{{stepNumber}}. Select File",
```

Comment:
Passing the step numbers as parameters seems overkill since they are fixed at compile time.

Reply:
Only reason is to avoid changing the `translation.json` file when adding a new step or something. I started with the numbers verbatim, looked at it and then thought it's going to be a nightmare.